GEOSldas_App/ldas_setup

```diff
@@ -168,8 +172,16 @@ class LDASsetup:
         self.ladas_coupling = int(self.rqdExeInp.get('LADAS_COUPLING',0))
         self.adas_expdir =''
         if self.ladas_coupling > 0:
-           assert 'ADAS_EXPDIR'  in self.rqdExeInp, " need ADAS_EXPDIR  in the input file %s" %(self.exeinpfile)
-           self.adas_expdir = self.rqdExeInp['ADAS_EXPDIR']
+           self.adas_expdir = os.path.dirname(self.exphome)
+           self.rqdExeInp[ 'ADAS_EXPDIR'] = self.adas_expdir
+           self.adas_expid = os.path.basename(self.adas_expdir)
+           self.rqdExeInp[ 'MET_TAG' ] = self.adas_expid + '__Nx+-'
+           if self.ladas_coupling == 1:
+                 self.rqdExeInp[ 'EXP_ID' ] = self.adas_expid + '_LDAS'
+           if self.ladas_coupling == 2:
+                 self.rqdExeInp[ 'EXP_ID' ] = self.adas_expid + '_LDAS4ens'
+                 self.rqdExeInp[ 'MET_PATH' ] = self.adas_expdir +'/atmens/ensdiag/mem'
```

Comment:
See comment above -- avoid adding "./ensdiag/" when we are already changing to "bkg"

Reply:
I will leave this in till we get bkg updates in and tested

Reply:
I guess the question is whether this PR needs to be merged before we can switch to the "bkg" convention for the lfo inputs to the  LDAS.  My preference would be to avoid merging this PR before we make the switch, but I'm not sure if that's workable